pages/common/ansible-galaxy.md

```diff
@@ -22,3 +22,7 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get the information for the role - or a collection if you replace `role` with `collection`:
+
+`ansible-galaxy role info {{username}}.{{role_name}}`
```

Comment:
How about we rephrase this a bit and split it into two examples? Also adresses @sbrl's suggestion above
```suggestion
- Get information about a role:

`ansible-galaxy role info {{username}}.{{role_name}}`

- Get information about a collection:

`ansible-galaxy collection info {{username}}.{{collection_name}}`
```
(I'm not sure if the second one is correct tough)

Reply:
Seems good